modules/visxBidAdapter.js

```diff
@@ -116,6 +117,24 @@ export const spec = {
 
       const { ortb2 } = bidderRequest;
       const { device, site, regs, content } = ortb2;
+      const userOrtb2 = ortb2.user;
+      let user;
+      let userReq;
+      const vads = _getUserId();
+      if (payloadUserEids || payload.gdpr_consent || vads) {
+        user = {
+          ext: {
+            ...(payloadUserEids && { eids: payloadUserEids }),
+            ...(payload.gdpr_consent && { consent: payload.gdpr_consent }),
+            ...(vads && { vads })
+          }
+        };
+      }
+      if (user) {
```

Comment:
please cover this behavior in a test so you can be confident it will not break. Thanks!